doc/install/stampede.tex

```diff
@@ -0,0 +1,157 @@
also refer to the user guide at: 
+
+http://www.tacc.utexas.edu/user-services/user-guides/stampede-user-guide
Also, please note that most parts of this document have changed since the previous
version. Even if you have set up ASPECT before, please follow all sections of this document
to make sure your current setup has all necessary updates.
+
+
+\section{Account}
+You first need to obtain an XSEDE account at http://portal.xsede.org/. If you need one send an email to cig-help@geodynamics.org.  
+
+\section{Setup}
+\begin{enumerate}
+\item To start edit your $\sim$/.bashrc file to load the correct modules. Update the file with the following in the 
+section that says PLACE MODULE COMMANDS HERE and ONLY HERE: \\
+
+  module load git \\
+  module load gcc/4.7.1 \\
+  module load mkl/13.0.2.146 \\
+  module load cmake \\
+
+
+\item Get trilinos on Stampede for compiling locally. 
+
+mkdir \$HOME/Downloads/ \\
+Download a copy of trilinos from trilinos.org. When creating this document we used 
+ trilinos-11.12.1-Source.tar because it's a known working version for the dealii-8.2.1 
+ version we install. \\
+ 
+ scp trilinos-11.12.1-Source.tar  loginname@stampede.tacc.utexas.edu:/work/02668/loginname/Downloads/  \\
+ 
+ \item Obtain and run currently working script for installing trilinos, p4est, and deal.ii  aspect-setup\_stampede.sh 
+ 
+ 
+Script is located in doc/install/aspect-setup\_stampede.sh \\
+ 
+ \end{enumerate}
+
+ \section{Install trilinos, p4est, and deal.ii}
+ 
+Since some of these libraries require a long time to compile, it may take a long time to
+finish if it is done on a login node (which is shared with dozens or hundreds of other users).
+It is recommended you log onto a compute node when compiling. To do so, use the following
+command: \\
+
+srun -p development -t 1:00:00 -n 16 --pty /bin/bash -l \\
+
+
+You will need to modify the script if you choose to install a different version of trilinos.  See comments within the script for additional information.\\
+ 
+ aspect-setup.sh trilinos \\
+ aspect-setup.sh p4est \\
+ aspect-setup.sh deal.ii    \\
+ 
+ You should now have the following directory structure in \$HOME/packages: \\
+  
+  build  deal.II  p4est  trilinos
+
+ \section{Configure and build ASPECT}
+ Here we assume you have a cloned version of ASPECT on github.  
+ cd \$HOME/packages \\
+ git clone https://github.com/your\_github\_account\_name/aspect.git \\
+ cd aspect \\
+ git remote add upstream https://github.com/geodynamics/aspect.git \\
+ git pull upstream master  \\
+ mkdir build \\
+ cd build \\
+ cmake ..  \\
+
+You will get a lot of warnings, but as long as you see the following you are ready to compile. \\
+-- Generating done \\
+-- Build files have been written to: /home1/02668/loginname/packages/aspect/build \\
+
+make -j8 \\
+cd ../ \\ 
+
+Check where you are located: \\
+git branch  \\
+
+You should see:  \\
+ $*$master  \\
+ 
+ Make a local branch for your work \\
+ git branch myproject \\
```

Comment:
I think the remainder of this file should not be in here (this is basic git usage and not related to stampede)